packages/date-picker/theme/lumo/vaadin-date-picker-overlay-content-styles.js

```diff
@@ -58,15 +56,8 @@ registerStyles(
       cursor: var(--lumo-clickable-cursor);
     }
 
-    [part='year-number']:not([current]),
-    [part='year-separator'] {
-      opacity: 0.7;
-      transition: 0.2s opacity;
-    }
-
-    [part='years']:hover [part='year-number'],
-    [part='years']:hover [part='year-separator'] {
-      opacity: 1;
+    ::slotted([slot='years']):hover {
+      --_lumo-date-picker-year-opacity: 1;
     }
```

Comment:
This selector isn't working properly, this seems to work instead:
```
::slotted([slot='years']:hover)
```

Reply:
Good catch, fixed.